app/views/system_admin/accredited_providers/confirmations/show.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="govuk-form-group">
```

Comment:
I'm not sure if this is the correct structure here. The form group adds unnecessary margin to the bottom. Throughout the app I think we've generally setup pages with the gov container classes then everything goes inside eg `govuk-grid-row` and `govuk-grid-column-two-thirds-from-desktop`